net/suricata6/Makefile

```diff
@@ -0,0 +1,112 @@
+#
+# Copyright (C) 2006-2015 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=suricata
+PKG_VERSION:=6.0.0-beta1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_DATE:=2020-10-08
+PKG_SOURCE_VERSION:=95729e923f39b5adda31e4843a1b6ee06edbb0c8
+PKG_SOURCE_URL:=https://github.com/OISF/suricata.git
+PKG_HASH:=skip
```

Comment:
Why? Please don't do this. If you are not sure which value put there, take a look at documentation or the build system can add the value for you.

Documentation here: https://openwrt.org/docs/guide-developer/packages#use_source_repository

Reply:
I am skipping the HASH because it's not needed for testing.  This will be completed with a FIXUP for a calc'ed SHA256 hash for the finalized .tar.xz file before I release it

Reply:
I think this is requirement otherwise Github Actions does not pass for this PR.